features/cellular/mbed_lib.json

```diff
@@ -10,7 +10,7 @@
             "value": false
         },
         "random_max_start_delay": {
-            "help": "Maximum random delay value used in start-up sequence in milliseconds",
+            "help": "Maximum random delay value used in start-up sequence in seconds",
```

Comment:
Another good catch 👍

Reply:
That's kind of the point of the Chrono stuff - it catches it for you :)